lib/fog/aws/models/elb/policy.rb

```diff
@@ -45,12 +44,8 @@ def destroy
           reload
         end
 
-        def reload
-          load_balancer.reload
-        end
-
         def load_balancer
-          collection.load_balancer
+          service.load_balancers.new(identity: collection.load_balancer_id)
```

Comment:
it's the last "," before the end of a Hash definition that is perhaps the biggest annoyance.  Quite possible the most understated Ruby improvement.